src/components/TeamCard/TeamCard.scss

```diff
@@ -6,8 +6,10 @@
   max-width: 292px;
   min-height: 392px;
   max-height: 500px;
-  margin-left: 10px;
-  margin-right: 10px;
+  margin: 3em 2.5em 0;
+  @media (max-width: 320px) {
+    margin: 3em 10px 0;
+  }
```

Comment:
I think on mobile we should have `margin:0; margin-bottom:50px` and on smallest the `margin: 3em 10px 0;` (we should be indicating all four margins instead of 3) and that seems to work nicely on mobile devices. Right now it's crunching it to scale similar to the galaxy fold but the screen width is making it extremely awkward. Also let's use the `responsiveness.screen(smallest)` instead of @media just so we are all adhering to the same viewport changes.